frontend/src/store/modules/plan.ts

```diff
@@ -2,10 +2,24 @@ import axios from "axios";
 import { Plan, FeatureType, PlanPatch, PlanState, PlanType } from "../../types";
 
 // A map from the a particular feature to the respective enablement of a particular plan
+// Make sure this is consistent with the matrix in plan.go
+//
+// TODO: fetch the matrix from the backend instead of duplicating it here or use a JSON/YAML file
+// so that it can be shared between frontend/backend. 
 const FEATURE_MATRIX: Map<FeatureType, boolean[]> = new Map([
```

Comment:
Shall we let the frontend fetch this from the backend? (I can implement this later in another PR)

Reply:
maybe a better way is to use a JSON file so that backend/frontend can also share it. It may be used in other places as well. wdyt?

If you like the approach, I may ask one of the interns to do this since it's a pretty good practice.

Reply:
Sure, LGTM, just like sharing the same config between backend and frontend.